src/gba/remote.cpp

```diff
@@ -2896,25 +2908,25 @@ void executeBreakCommands(int n, char** cmd)
         operation(address, flag, cmd + 1, n - 1);
         return;
     }
-
-brkcmd_special_register:
-    switch (command[4]) {
-    case 'l':
-        debuggerBreakRegisterList((n > 0) && (tolower(cmd[0][0]) == 'v'));
-        return;
-    case 'c':
-        debuggerBreakRegisterClear(n, cmd);
-        return;
-    case 'd':
-        debuggerBreakRegisterDelete(n, cmd);
-        return;
-
-    case 'm':
-        debuggerBreakRegister(n, cmd);
-    default:
-        return;
-    };
-    return;
+// Not used anywhere...
```

Comment:
I'm not sure, the first else above does not `return`.

Reply:
This is a `goto` scope. It needs to be called, right? The only match when running:
`fgrep -Rnw . -e 'brkcmd_special_register'` was this piece of declaration.

Reply:
The label does not prevent the code from executing, you can `goto` it explicitly, but it will also be executed if the the function does not `return` beforehand.